File: cfgs/java_cfgs_foo/FIND_SUM_MODULO_K_FIRST_N_NATURAL_NUMBER_1.java
Nodes
1;special;;;
2;statement;int ans = 0;14;
3;statement;int y = foo_1 / foo_2;15;
4;statement;int x = foo_1 % foo_2;16;
5;statement;ans = (((foo_1 * (foo_2 - 1)) / 2) * foo_3) + ((foo_4 * (foo_5 + 1)) / 2);17;
6;statement;return foo_1;18;
7;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;
4;6;
5;7;
6;

File: cfgs/pythoncfgs/FIND_SUM_MODULO_K_FIRST_N_NATURAL_NUMBER_1.py
Nodes
1;statement;foo_1 = 0;8;
2;statement;foo_1 = foo_2 / foo_3;9;
3;statement;foo_1 = foo_2 % foo_3;10;
4;statement;foo_1 = foo_2 * (foo_3 - 1) / 2 * foo_4 + foo_5 * (foo_6 + 1) / 2;11;
5;statement;return fun_1(foo_1);12;
6;exit;;
Edges
0;2;
1;3;
2;4;
3;5;
4;6;
5;
